lib/Doctrine/ODM/PHPCR/Translation/TranslationStrategy/AttributeTranslationStrategy.php

```diff
@@ -173,9 +173,9 @@ public function removeAllTranslations($document, NodeInterface $node, ClassMetad
     public function getLocalesFor($document, NodeInterface $node, ClassMetadata $metadata)
     {
         $locales = array();
-        foreach ($node->getProperties("*{$this->prefix}*") as $prop) {
+        foreach ($node->getProperties($this->prefix . ':*') as $prop) {
             $matches = null;
-            if (preg_match('/' . $this->prefix . ':(..)-[^-]*/', $prop->getName(), $matches)) {
+            if (preg_match('/' . $this->prefix . ':([a-zA-Z1-9_]+)-[^-]*/', $prop->getName(), $matches)) {
```

Comment:
ach. howto regex. actually this does what it should. only the `[^-]` part is useless. will remove it.